package.json

```diff
@@ -8,24 +10,24 @@
     "postinstall": "jetify"
   },
   "dependencies": {
-    "@react-native-community/geolocation": "^2.0.2",
+    "@react-native-community/geolocation": "^2.1.0",
     "@react-native-community/push-notification-ios": "^1.10.1",
-    "appcenter": "4.4.3",
-    "appcenter-crashes": "4.4.3",
-    "bluedot-react-native": "^2.1.0",
-    "expo": ">=44.0.0-0 <45.0.0",
-    "expo-splash-screen": "~0.14.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-native": "0.67.3",
-    "react-native-gesture-handler": "~2.1.0",
-    "react-native-permissions": "^3.3.1",
-    "react-native-push-notification": "^8.1.1",
-    "react-native-reanimated": "~2.3.1",
-    "react-native-screens": "~3.10.1",
-    "react-native-web": "0.17.1",
-    "react-router-dom": "^6.2.2",
-    "react-router-native": "^6.2.2"
+    "appcenter": "4.4.5",
+    "appcenter-crashes": "4.4.5",
+    "bluedot-react-native": "Bluedot-Innovation/Bluedot-React-Native-Plugin#dn/react-native-0.69",
```

Comment:
pointing to 0.69 branch of plugin repo for QA. Will point to official version 2.1.3 when it's done QA and published